### ISSUE TITLE
Truncate the /tools section

### ISSUE THREAD

**Issue by bep** [gohugoio/hugoDocs#1573]
I had a look at the tools section

https://gohugo.io/tools/

And that needs some gardening. Looking through it, 1) I have never heard of 90% of the items 2) I suspect many of them aren't "notable enough".

I understand that people want a link to their project they have hard on, but we cannot be a link collection.

I suggest that we start a little fresh -- make "Tools" a one-pager and remove everything but the obvious (not sure what that should be, but I can certainly try if we agree on this).

/cc @jmooring

**Comment by coliff** [comment]
I agree! I think instead it could be useful to include a link to something like:
https://github.com/topics/hugo

That way users can see all projects tagged with Hugo and sort by Best match, most stars, most recently updated etc.

**Comment by danielfdickinson** [comment]
I agree the Tools page is better kept short. As I commented to @jmooring in [#1512#issuecomment-1079740244](https://github.com/gohugoio/hugoDocs/pull/1512#issuecomment-1079740244) I think that there should be a pointer to a 'wiki' post or a discussion on the [Hugo forum](https://discourse.gohugo.io/).

I agree with him that this is better done as full HOWTOs elsewhere and the forum seems the best place to me.

**Comment by jmooring** [comment]
Also move https://gohugo.io/content-management/comments/ to the tools section, or whatever it will called once trancated and regorganized. The Disqus setup is referenced elsewhere in the documentation.
